File: LeaguerManagement.Entities/ViewModels/Settings/HolidayModel.cs
using System;

namespace LeaguerManagement.Entities.ViewModels.Settings
{
    public class HolidayModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime Date { get; set; }
        public bool IsSaturday { get; set; }
        public bool IsSunday { get; set; }
        public bool Is11 { get; set; }
        public bool Is304 { get; set; }
        public bool Is15 { get; set; }
        public bool Is29 { get; set; }
        public bool IsSettings { get; set; }
        public int Year { get; set; }
    }
}

File: LeaguerManagement.UI/src/app/models/settings/holiday.model.ts
export class HolidayModel {
  id: number;
  name: string;
  year: number | null;
  date: Date;
  isSaturday: boolean = false;
  isSunday: boolean = false;
  is11: boolean = false;
  is304: boolean = false;
  is15: boolean = false;
  is29: boolean = false;
  isSettings: boolean = false;

  constructor(init?: Partial<HolidayModel>) {
    Object.assign(this, init);
  }
}
